### PR TITLE
Support reading hl7v2 parser config from file.

### PR DESCRIPTION
**Pull request by fbeevikm**
Deployed the change in HDE instance for testing.
instance prefix : test-dpt1
Verified that the parser config through schema_file has been applied to hl7 store successfully.
https://screenshot.googleplex.com/8NJfhrimp44YaKB.png

### CODE REVIEW

File: templates/tfengine/recipes/resources.hcl [hunk 1]
```diff
@@ -1108,6 +1108,10 @@ schema = {
                     schema = {
                       type = "string"
                     }
+                    schema_file = {
+                      description = "Path to the JSON encoded schema file."
```
Comment (yeweidaniel) [inline templates/tfengine/recipes/resources.hcl:1112]: Mention that either schema or schema_file should be provided, not both.

File: examples/tfengine/modules/team.hcl [hunk 2]
```diff
@@ -330,7 +330,10 @@ EOF
     }
   }
 }
-
+template "schema" {
+  component_path = "./schema/hl7_config_schema.json"
+  output_path = "./project_data/schema.json"
```
Comment (yeweidaniel) [inline examples/tfengine/modules/team.hcl:335]: nit: keep the file name the same to avoid confusion

**Review by yeweidaniel — APPROVED** [review]

File: examples/tfengine/generated/team/project_data/schema.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "schematizedParsingType": "SOFT_FAIL",
+  "ignoreMinOccurs": true
+}
```
Comment (xingao267) [inline examples/tfengine/generated/team/project_data/schema.json:4]: add an empty line at the end of file

File: examples/tfengine/modules/schema/hl7_config_schema.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "schematizedParsingType": "SOFT_FAIL",
+  "ignoreMinOccurs": true
+}
```
Comment (xingao267) [inline examples/tfengine/modules/schema/hl7_config_schema.json:4]: add an empty line at the end of file

File: .gitignore [hunk 5]
```diff
@@ -23,3 +23,5 @@
 # Terraform state
 *.tfstate
 *.tfstate.*
+
+.idea/
```
Comment (xingao267) [inline .gitignore:27]: Add a comment for this.

**Review by xingao267 — APPROVED** [review]

**Comment by xingao267** [comment]
After the comment in `.gitignore` is added, please trigger the integration tests by writting `/gcbrun` in a comment in the Pull Request.

**Pull request merged**